### PR TITLE
ceph-volume: add missing scenarios to nightlies

### PR DESCRIPTION
**Pull request by rishabh-d-dave**

### CODE REVIEW

**Comment by rishabh-d-dave** [comment]
About last push: added a missing LVM scenario.

File: ceph-volume-nightly/config/definitions/ceph-volume-nightly.yml [hunk 1]
```diff
@@ -10,6 +10,7 @@
     scenario:
       - create
       - dmcrypt
+      - prepare_activate
```
Comment (jan--f) [inline ceph-volume-nightly/config/definitions/ceph-volume-nightly.yml]: Where is this from?

Comment (rishabh-d-dave) [reply ceph-volume-nightly/config/definitions/ceph-volume-nightly.yml]:
```
$ tox -l
centos8-filestore-create
centos8-filestore-prepare_activate
centos8-filestore-dmcrypt
centos8-bluestore-create
centos8-bluestore-prepare_activate
centos8-bluestore-dmcrypt
$ pwd
/home/fedora/ceph/src/ceph-volume/ceph_volume/tests/functional/lvm
```

Comment (jan--f) [reply ceph-volume-nightly/config/definitions/ceph-volume-nightly.yml]: but
```
% l tests/functional/lvm/centos8/bluestore                                                                                              :(
total 0
drwxr-xr-x 1 jan jan 128 Feb 25 16:46 create
drwxr-xr-x 1 jan jan 128 Feb 25 16:46 dmcrypt
```
seems to be the tox.ini needs fixing.

Comment (jan--f) [reply ceph-volume-nightly/config/definitions/ceph-volume-nightly.yml]: in any case, create is only a convenience command for prepare and activate.

Comment (rishabh-d-dave) [reply ceph-volume-nightly/config/definitions/ceph-volume-nightly.yml]: Okay, got rid of it.

**Pull request merged**